Frontend/ui-customer/src/Flow.purs

```diff
@@ -425,6 +425,7 @@ homeScreenFlow = do
         modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen{props{currentStage = SearchLocationModel ,rideRequestFlow = false, isSearchLocation = SearchLocation, isSrcServiceable = false, isSource = Just true, isRideServiceable = false}})
         homeScreenFlow
         else pure unit
+      _ <- pure $ setValueToLocalStore UNIQUE_ID (generateUId unit)
```

Comment:
I think we are doing the `/rideSearch` api in two places in Flow file update the `session_id` in all places